tests/test_client.py

```diff
@@ -147,3 +153,225 @@ def on_message(client, userdata, msg):
 
         packet_in = fake_broker.receive_packet(1)
         assert not packet_in  # Check connection is closed
+
+
+@pytest.mark.parametrize("proto_ver,proto_name", [
+    (client.MQTTv31, "MQIsdp"),
+    (client.MQTTv311, "MQTT"),
+])
+class TestInvalidWebsocketResponse(object):
```

Comment:
I wonder if it's better to put these websocket tests into a different file. This one is mostly about moving old tests to py.test.